src/amo/pages/Home/styles.scss

```diff
@@ -1,7 +1,17 @@
 @import '~amo/css/styles';
 
 .Home {
-  @include page-padding();
+  width: 100%;
```

Comment:
I don't know for sure if this is the culprit, but there's a regression in the loading transition between pages. You can see it browsing the site in a large screen and navigating from home to a sub-page. Here is a screenshot using Redux devtools so you can see it slow motion:

<details>
<summary>Screenshot</summary>

![2019-10-01 09 43 23](https://user-images.githubusercontent.com/55398/65972981-3af1d080-e430-11e9-9a33-b4aab4558921.gif)


</details>

Reply:
The problem is that, when switching from the home page to another page, the `viewContext` remains `VIEW_CONTEXT_HOME` when the page is first rendered, and that is what is causing the layout problem. I fixed this by making sure that the `viewContext` resets whenever the home page is navigated away from, and I had to make sure this only happens on the client or it causes a rendering glitch on the server. I believe this fix is okay, and didn't detect an regressions as a result, but I'm keen to see what you think about it.